app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -13,8 +13,14 @@ def initialize(enrollment:, barcode_image_url: nil)
         @barcode_image_url = barcode_image_url
       end
 
+      # Reminder of days left is exclusive of the day the email is sent, hence the reminder num is 1 less than days_to_due_date
+      def days_reminder
+        return 3 if enrollment.days_to_due_date == 4
+        return 10 if enrollment.days_to_due_date == 11
```

Comment:
could we set make this do subtraction? That way if we change the schedule elsewhere, we don't need to come back and add another branch here?
```suggestion
        enrollment.days_to_due_date - 1
```

Reply:
i did think of doing that but also was thinking we dont want to display any days if we're not at those specific markers.

Reply:
My expectation would be that the logic for determining whether a message should be sent would live elsewhere, and the logic here would return based on the days remaining, regardless if it's sent on one of the dates we're expecting.

Also captured as a relevant note in the ticket:

>Let's include the ability to make the number dynamic- 10 and 3 are guesses [...] Ex: if the email was not able to be sent on day 3, we will dynamically change it so it reflects the day it is able to be sent